introduction.md

```diff
@@ -0,0 +1 @@
+Hi. My name is Perry Thomson.  I am learning 'git' to improve my performance.  My particular interests are in javascript, java, and python.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.